### PR TITLE
Update tasks/eth/1-security-council-phase-0

### PR DESCRIPTION
**Pull request by zchn**
Not all council members have passed rehearsal 4, but I've verified that all 13 of them passed rehearsal 2, so I'm doing some branch prediction and creating the safe to use proactively here.

Also, I did not successfully run `just simulate` locally yet, but the simulation from the CI/CD works: https://dashboard.tenderly.co/k-oplabs/project/simulator/31dfb05e-084a-4a3c-a6b6-3b7a96146e94/state-diff

I will do another PR to use a new Foundation multisig if we get the sufficient approval to do so.

I will mark the playbook as READY_TO_SIGN when
1. all members pass the rehearsal 4
2. foundation has published the official signer addresses and threshold in the gov forum.

### CODE REVIEW

File: tasks/eth/1-security-council-phase-0/README.md [hunk 1]
```diff
@@ -83,23 +83,37 @@ validate integrity of the simulation, we need to check the following:
...
+   `0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`:
+
+   a. One of them overrides storage slot `0x4` to new value
+   `0x1`. This override is only intended to change the Foundation
+   multisig's quorum threshold to 1 so we can perform a tenderly
+   simulation of the execution,
+
+   b. One of them overrides storage slot `0x5` to `0x56`. This
+   override is an no-op to override the nonce of the multisig to 86,
+   which is the same as it's current value.
```
Comment (mds1) [inline tasks/eth/1-security-council-phase-0/README.md:96]: Why does this no-op override exist? I'll also note that I've confirmed this is a no-op by running `cast call 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A "nonce()(uint256)" --rpc-url $MAINNET_RPC_URL`, which might be nice to add here

File: tasks/eth/1-security-council-phase-0/README.md [hunk 2]
```diff
@@ -3,19 +3,19 @@
...
 
 This is the playbook for executing the Security Council Phase 0 as
-approved by Governance. There are two governance proposals related to
-this:
+approved by Governance. There are two governance proposals and one
+forum update related to this:
 
 1. [Security Council: Vote #1](https://vote.optimism.io/proposals/27439950952007920118525230291344523079212068327713298769307857575418374325849).
 2. [Security Council Membership Ratification](https://vote.optimism.io/proposals/85591583404433237270543189567126336043697987369929953414380041066767718361144).
+3. [Governance Post containing the threshold and signer addresses](https://gov.optimism.io/t/security-council-vote-2-initial-member-ratification/7118/19)
```
Comment (mds1) [inline tasks/eth/1-security-council-phase-0/README.md:16]: nit, changes the link to go to a comment from "system" confirming that 4/14 is expected threshold for now
```suggestion
3. [Governance Post containing the threshold and signer addresses](https://gov.optimism.io/t/security-council-vote-2-initial-member-ratification/7118/13)
```

File: tasks/eth/1-security-council-phase-0/README.md [hunk 3]
```diff
@@ -83,23 +83,37 @@ validate integrity of the simulation, we need to check the following:
...
+   a. Verify that the `_owner` of `ProxyAdmin` is changed to a 2 of 2
+   multisig.
+
+   b. Verify that the 2 signers are the Foundation multisig
+   (`eth:0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`) and the Security
+   Council multisig
+   (`eth:0xc2819DC788505Aac350142A7A707BF9D03E3Bd03`).
+
+   c. Verify that the Security Council multisig's threshold and
+   signers match what's published in the governacne forum.
```
Comment (mds1) [inline tasks/eth/1-security-council-phase-0/README.md:113]:
```suggestion
   signers match what's published in the governance forum.
```

File: tasks/eth/1-security-council-phase-0/input.json [hunk 4]
```diff
@@ -10,7 +10,7 @@
     {
       "to": "0x543bA4AADBAb8f9025686Bd03993043599c6fB04",
       "value": "0",
-      "data": "0xf2fde38b000000000000000000000000dc53b1d440b8a56dd50a6d69312dac903d3aa48b",
+      "data": "0xf2fde38b00000000000000000000000027d3C6Ac002fc5B5fAcf44cAa1C63d978b4A7e60",
```
Comment (mds1) [inline tasks/eth/1-security-council-phase-0/input.json:13]: I ran:
```
$ cast 4byte-decode 0xf2fde38b00000000000000000000000027d3C6Ac002fc5B5fAcf44cAa1C63d978b4A7e60
1) "transferOwnership(address)"
0x27d3C6Ac002fc5B5fAcf44cAa1C63d978b4A7e60
```
to verify that the data here matches what's shown in the simulation. Will defer to you about whether this is worth adding as a validation step

File: tasks/eth/1-security-council-phase-0/README.md [hunk 5]
```diff
@@ -83,23 +83,37 @@ validate integrity of the simulation, we need to check the following:
 
 Now click on the "State" tab. Verify that:
 
-1. There is only a single state override at address
-   `0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`, which overrides
-   storage slot `0x4` to new value `0x1`. This override is only
-   intended to change the Foundation multisig's quorum threshold to 1
-   so we can perform a tenderly simulation of the execution,
+1. There are only two single state override at address
```
Comment (maurelian) [inline tasks/eth/1-security-council-phase-0/README.md:86]:
```suggestion
1. There are only two state overrides at address
```

**Review by mds1 — APPROVED** [review]

**Pull request merged**